recipes/quickfix/all/conanfile.py

```diff
@@ -11,8 +12,15 @@ class QuickfixConan(ConanFile):
     topics = ("conan", "QuickFIX", "FIX", "Financial Information Exchange", "libraries", "cpp")
     settings = "os", "compiler", "build_type", "arch"
     options = {"fPIC": [True, False],
-               "ssl":  [True, False]}
-    default_options = {"ssl": False, "fPIC": True}
+               "shared": [True, False],
+               "with_ssl":  [True, False],
+               "with_postgres": [True, False],
+               "with_mysql": [None, "libmysqlclient"]}
```

Comment:
Why not True/False? Is there a third option?

Reply:
I've made this way following gdal's conanfile.py example:

https://github.com/conan-io/conan-center-index/blob/345f7617b97a1fb55d4744c91e8f131032d32001/recipes/gdal/all/conanfile.py#L46-L49

I thought it would be a good idea to support other mysql recipes in the future, as there's already another recipe for mysql (I assume that this is what mysql-connector-c is, but I really don't know the difference between those two).

Maybe I'm again complicating things prematurely?

Reply:
I see, in this case, keep it. Changing from bool to string can break the users in the future. Your approach is correct, thanks for explaining.